internal/controllers/clusterextension_controller_test.go

```diff
@@ -21,191 +21,191 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/client/fake"
 
-	operatorsv1alpha1 "github.com/operator-framework/operator-controller/api/v1alpha1"
+	ocv1alpha1 "github.com/operator-framework/operator-controller/api/v1alpha1"
 	"github.com/operator-framework/operator-controller/internal/catalogmetadata"
 	"github.com/operator-framework/operator-controller/internal/conditionsets"
 	"github.com/operator-framework/operator-controller/pkg/features"
 )
 
-// Describe: Operator Controller Test
-func TestOperatorDoesNotExist(t *testing.T) {
+// Describe: ClusterExtension Controller Test
+func TestClusterExtensionDoesNotExist(t *testing.T) {
 	_, reconciler := newClientAndReconciler(t)
 
-	t.Log("When the operator does not exist")
+	t.Log("When the cluster extension does not exist")
 	t.Log("It returns no error")
 	res, err := reconciler.Reconcile(context.Background(), ctrl.Request{NamespacedName: types.NamespacedName{Name: "non-existent"}})
 	require.Equal(t, ctrl.Result{}, res)
 	require.NoError(t, err)
 }
 
-func TestOperatorNonExistantPackage(t *testing.T) {
+func TestClusterExtensionNonExistantPackage(t *testing.T) {
```

Comment:
Since this is being updated might as well fix the spelling error :):
```suggestion
func TestClusterExtensionNonExistentPackage(t *testing.T) {
```